src/assets/styles/components/_navigation.css

```diff
@@ -55,40 +55,22 @@ a[rel="home"] {
     }
 }
 
-.navigation__toggle {
+.navigation button.borderless {
```

Comment:
Because of the class name change in menu button in navigation bar, the button is displayed even on large screens, and that's because there is responsive rule specified for .navigation__toggle in line 184 of this file. We need to update line 184 to:

```
.navigation button.borderless {
    display: none;
}

```